shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/Coin.hs

```diff
@@ -58,6 +59,11 @@ addDelta (Coin x) (DeltaCoin y) = Coin (x + y)
 toDelta :: Coin -> DeltaCoin
 toDelta (Coin x) = DeltaCoin x
 
+instance Torsor.Torsor Coin where
+  type Delta Coin = DeltaCoin
+  addDelta = addDelta
+  toDelta = toDelta
```

Comment:
There's a danger of the following happening in modules that import both:
```
    Ambiguous occurrence ‘toDelta’
    It could refer to
       either ‘Cardano.Ledger.Torsor.toDelta’,
              imported from ‘Cardano.Ledger.Torsor’
           or ‘Shelley.Spec.Ledger.Coin.toDelta’,
              imported from ‘Shelley.Spec.Ledger.Coin’
```
`toDelta :: Coin -> DeltaCoin` could be renamed to `toDeltaCoin`?